### PR TITLE
Add Sports Score Metadata

### PR DESCRIPTION
**Pull request by itsibitzi**
Designs from Ana P require a little yellow box for sports scores - not dissimilar to how we display review ratings currently. 

This PR adds that field as part of the metadata on a given article.

### CODE REVIEW

File: test/editions/PublishedIssueTest.scala [hunk 1]
```diff
@@ -31,6 +31,7 @@ class PublishedIssueTest extends FreeSpec with Matchers {
         Some(true),
         Some(true),
         Some("byline"),
+        None,
```
Comment (sihil) [inline test/editions/PublishedIssueTest.scala]: This test is for when the furniture is populated in order to ensure that it gets through to Alex. Can you please update to ensure a value is passed through?

**Comment by itsibitzi** [comment]
image

Screenshot of the input

File: test/editions/PublishedIssueTest.scala (hunk 1 above)
Comment (itsibitzi) [reply test/editions/PublishedIssueTest.scala]: Done!

File: client-v2/src/selectors/formSelectors.ts [hunk 2]
```diff
@@ -66,6 +66,13 @@ export const createSelectFormFieldsForCollectionItem = () => {
       }
       const fields = defaultFields.slice();
 
+      if (
+        derivedArticle.sectionName === 'Sport' ||
+        derivedArticle.sectionName === 'Football'
+      ) {
+        fields.push('sportScore');
```
[the remark this answers is not in the document]
Comment (itsibitzi) [reply client-v2/src/selectors/formSelectors.ts]: Yes. Sloppy mistake, sorry.

I've approved your PR! :)

[the remark this answers is not in the document]
Comment (akash1810) [reply client-v2/src/selectors/formSelectors.ts]: minor: could we either push this down to line 90 or push line 90 up as a way to only do the `if (editMode === 'editions')` check once

[the remark this answers is not in the document]
Comment (itsibitzi) [reply client-v2/src/selectors/formSelectors.ts]: Yeppers - done!

**Pull request merged**